File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.cs
using System;

public class Solution 
{
    public int solution(int[] num_list) 
    {
        int i = 0;
        int count = 0;
        
        foreach(int num in num_list) {
            int current = num;
             while(current > 1) {
            
                if(current%2 == 0) {
                    current /= 2;
                    count++;
                }
                else if(current%2 == 1) {
                    current -= 1;
                    current /= 2;
                    count++;
                }
            }
        }      
        return count;
    }
}

File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) 
{
    int count = 0;
    for(int n : num_list)
    {
        while(n != 1)
        {
            if(n%2 == 0)
            {
                n /= 2;
                count++;
            }
            else
            {
                n -= 1;
                n /= 2;
                count++;
            }     
        }
    }
    return count;
}